apps/web/components/eventtype/EventLimitsTab.tsx

```diff
@@ -88,6 +88,7 @@ const MinimumBookingNoticeInput = React.forwardRef<
           label={t("minimum_booking_notice")}
           type="number"
           placeholder="0"
+          min={0}
```

Comment:
For minimum bookings, they should never be below zero. Ex -2

Reply:
Can we validate this on server as well?

Reply:
But that's not enough if we're not actually using the schema in de endpoint. Are we?